guides/common/modules/proc_customizing-registration-by-using-snippets.adoc

```diff
@@ -0,0 +1,23 @@
+[id="customizing-registration-by-using-snippets"]
+= Customizing registration by using snippets
+
+You can customize the registration process without editing the *Global Registration* template by creating snippets that the template includes if they exist.
```

Comment:
```suggestion
You can customize the registration process without editing the `Global Registration` template by creating snippets that the template automatically includes if they exist.
```

alternative to "automatically": "by default, {Project} includes...".

Reply:
The part with `if they exist` at the end tripped me up :) Would the sentence work without it? (I think it would.) Alternatively, how about this?

```suggestion
You can customize the registration process by creating snippets for the `Global Registration` template. The template includes the snippets automatically if they exist. This does not require editing the template.
```

This is much longer, but by presenting the information in multiple shorter sentences (one fact = one sentence) reduces the risk of users having to re-read.